docs/labs.md

```diff
@@ -130,6 +130,12 @@ encrypted on the server, as described in [MSC2697](https://github.com/matrix-org
 Enables UI for turning on "do not disturb" mode for the current device. When DND mode is engaged, popups
 and notification noises are suppressed. Not perfect, but can help reduce noise.
 
+## Giphy Integration (`feature_giphy_integration`)
+
+Giphy API integration, that allows the users to send GIFs by searching with keywords, and selecting the one they like. It displays a new button in the composer next to the stickers and emojis. The GIFs are sent as files/attachments in the conversation.
+
+Associated PR: https://github.com/matrix-org/matrix-react-sdk/pull/5814
```

Comment:
I don't think we usually link to a specific PR. You might do some follow up PRs and it doesn't really make sense to link them all here. What we usually do though is link the PR in this PR's description